types/c3/index.d.ts

```diff
@@ -1,26 +1,57 @@
-// Type definitions for C3js 0.7
+// Type definitions for C3js 0.7.2
 // Project: http://c3js.org/, https://github.com/c3js/c3
 // Definitions by: Marc Climent <https://github.com/mcliment>
 //                 Gerin Jacob <https://github.com/gerinjacob>
 //                 Bernd Hacker <https://github.com/denyo>
 //                 Dzmitry Shyndzin <https://github.com/dmitryshindin>
 //                 Tim Niemueller <https://github.com/timn>
 //                 Nate Mara <https://github.com/natemara>
+//                 Ian Sanders <https://github.com/iansan5653>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 3.5.2
 
 import * as d3 from 'd3';
 
 export as namespace c3;
 
-export type PrimitiveArray = Array<string | boolean | number | null>;
-export type FormatFunction = (v: any, id: string, i: number, j: number) => void;
+export function generate(config: ChartConfiguration): ChartAPI;
 
-export interface TargetIds {
-    ids: ArrayOrString;
-}
+export const version: string;
 
+export type Primitive = string | boolean | number | null;
+export type PrimitiveArray = Array<string | boolean | number | null>;
```

Comment:
Should use `Primitive` from the previous line.

Reply:
Fixed with the latest commits.